airflow-core/src/airflow/api_fastapi/core_api/datamodels/dag_run.py

```diff
@@ -74,6 +74,7 @@ class DAGRunResponse(BaseModel):
     run_type: DagRunType
     state: DagRunState
     triggered_by: DagRunTriggeredByType | None
+    triggering_user: str | None
```

Comment:
triggered_by feels like a good field to use to track "who"(user) triggered it.. and for type, we could have used "trigger_type".

Reply:
Yeah, naturally `triggered_by` would also be my favorite but it was used before. In order to prevent confusion I'd prefer not to change field definitions. `trigger_type` would be a bit confusing to `run_type`

Reply:
Hi, I was the one who added the `triggered_by` field (1 year ago). At that time I was not aware/expect that we will add a user relation here. But now, when I see this PR; I wish I selected a different name 😅 

Maybe we can rename the `triggered_by` field as `triggered_with` (or `trigger_source`) and use `triggered_by` as a user reference. Ofc, if the other reviewers think the same